### PR TITLE
Clean up code block, correct variable name

### PR DESCRIPTION
**Pull request by gmal1**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/russian/06-information-security-and-quality-assurance/advanced-node-and-express/announce-new-users.russian.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ localeTitle: Объявление новых пользователей
...
   var message = data.name;
   if (data.connected) {
-    сообщение + = &#39;присоединилось к чату.&#39;;
+    message += &#39;присоединился(лась) к чату.&#39;;
   } else {
-    сообщение + = &#39;покидает чат.&#39;;
+    message += &#39;покинул(а) чат.&#39;;
   }
-  $ (&#39;# messages&#39;). append ($ (&#39;&lt;li&gt;&#39;). html (&#39;&lt;b&gt;&#39; + message + &#39;&lt;\ / b&gt;&#39;));
+  $ (&#39;#messages&#39;).append($(&#39;&lt;li&gt;&#39;).html(&#39;&lt;b&gt;&#39; + message + &#39;&lt;\/b&gt;&#39;));
```
Comment (thecodingaviator) [inline curriculum/challenges/russian/06-information-security-and-quality-assurance/advanced-node-and-express/announce-new-users.russian.md]:
```suggestion
  $(&#39;#messages&#39;).append($(&#39;&lt;li&gt;&#39;).html(&#39;&lt;b&gt;&#39; + message + &#39;&lt;\/b&gt;&#39;));
```

**Review by thecodingaviator — DISMISSED** [review]

**Comment by RandellDawson** [comment]
@raisedadead How can we start the workflow build?

**Comment by thecodingaviator** [comment]
@RandellDawson I think that since we didn't have a CircleCI config as of then, we can skip that test for now. Stuart(Bouncey) is working on one.

**Review by greggubarev — APPROVED** [review]
LGTM 👍 :dart:

Thank you for the PR ✨ 🎉
 We're happy to accept these changes and look forward to your future contributions :clap:

Happy coding! :octocat:

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**